lib/utils.js

```diff
@@ -11,31 +20,14 @@ const severity = {
   debug: 7
 }
 
-function levelToSeverity (level) {
-  let result
-  switch (level) {
-    case 10: // pino: trace
-      result = severity.debug
-      break
-    case 20: // pino: debug
-      result = severity.debug
-      break
-    case 30: // pino: info
-      result = severity.info
-      break
-    case 40: // pino: warn
-      result = severity.warning
-      break
-    case 50: // pino: error
-      result = severity.error
-      break
-    case 60: // pino: fatal
-    default:
-      result = severity.critical
-      break
-  }
-  return result
-}
+const pinoLevelToSyslogSeverity = new Map([
+  [pinoLevels.trace, severity.debug],
+  [pinoLevels.debug, severity.debug],
+  [pinoLevels.info, severity.info],
+  [pinoLevels.warn, severity.warning],
+  [pinoLevels.error, severity.error],
+  [pinoLevels.fatal, severity.critical]
+])
```

Comment:
This is not even close to as legible as the replaced code. If we are going to go this direction, there needs to be a jsdoc block that clearly documents what the map structure is and how it is to be used.